harness/determined/common/storage/azure_client.py

```diff
@@ -0,0 +1,67 @@
+import logging
```

Comment:
should I hide this by naming it `_azure_client.py`?

Reply:
looks like surrounding files aren't "underscored", so this should be ok.

Reply:
As @ioga said, the way it is named is fine due to the legacy files around it.

But if you are looking for some rules on naming conventions that we have applied to newer or more-user-facing parts of the python code, here is the doc: https://github.com/determined-ai/pedl/pull/4727